configs/.hmy/keystore/0x0DdC7838d604B00210Ce1e17369D22b26058Fe01.key

```diff
@@ -0,0 +1 @@
+{"address":"0ddc7838d604b00210ce1e17369d22b26058fe01","crypto":{"cipher":"aes-128-ctr","ciphertext":"42a53016e6fe135b702857cecd580022daf8e238a254c2cbc5824e356ffcc8cf","cipherparams":{"iv":"de38df0388f47858ad7cb6633ec583a5"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"f5ad04124588ab38712f3d48c6c83a3c28dcc709acca673cc7d6520456659284"},"mac":"9c00da26a5d9cf54faeff082bbfd4f989cae4da6b5fb05a76388d96e0dbe4fea"},"id":"77606bf2-62ab-4a33-b2f0-28c00f220d9e","version":3}
```

Comment:
What are these key files? our own nodes?

Reply:
yes. for local testing.

Reply:
those are for testing now.  for mainnet, we need to encrypt the key as well.